docs/network/monitoring.md

```diff
@@ -0,0 +1,314 @@
+
+This table shows the available labels:
+
+|             Metric             |        Label(s)        |                       Description                         |
+|:------------------------------:|:----------------------:|:---------------------------------------------------------:|
+| information                    | commit_hash            | commit hash of the node version                           |
+| information                    | commit_date            | commit date of the node version                           |
+| information                    | mode                   | mode of the node (RPC, observer, sequencer, proxy)        |
+| information                    | smart_rollup_address   | address of the smartrollup monitored                      |
+| calls                          | endpoint               | endpoint reached (`/`, etc.)                              |
+| calls                          | method                 | JSON RPC method called (`etc_call`, etc.)                 |
+| calls_method                   | method                 | JSON RPC method called (`etc_call`, etc.)                 |
+| tx_pool                        | kind                   | Either `number_of_addresses` of `number_of_transactions`  |
+
+### Scraping metrics with prometheus
+
+Scraping metrics gives you instant values of the metrics. For a more effective monitoring, you should create a time series of these metrics.
+
+We suggest using [Prometheus](https://prometheus.io/) for that purpose.
+
+First, you need to install Prometheus from your favourite package manager, e.g.
+
+```bash
+sudo apt install prometheus
+```
+
+Once installed, you need to add the scraping job to the configuration file (e.g. ``/etc/prometheus/prometheus.yml``)
+
+```yaml
+- job_name: 'octez-evm-node-exporter'
+  scrape_interval: <interval> s
+  metrics_path: "/metrics"
+  static_configs:
+    - targets: ['<addr>:<port>']
+```
+
+For instance, a typical target address can be ``localhost:8545``.
+
+If you plan to also use Prometheus Node Exporter (as described [below](#hardware-metrics), you should also add:
+
+```yaml
+- job_name: 'node-exporter'
+  scrape_interval: <interval> s
+  static_configs:
+    - targets: ['<addr>:<port>']
+```
+
+For instance, a typical target address may be ``localhost:9100``.
+
+Note that Prometheus can also scrape metrics from several nodes.
+
+Prometheus is a service, so you need to start it:
+
+```bash
+sudo systemctl start prometheus
+```
+
+You may visualize the log in real time to make sure the service works correctly
+
+```bash
+sudo journalctl --follow --unit=prometheus.service
+```
+
+#### History size
+
+By default, Prometheus keeps 15 days of data. It may be useful to increase the history window to keep wider history. To do so, update the Prometheus service execution argument (typically, in ``/etc/systemd/system/multi-user.target.wants/prometheus.service``).
```

Comment:
```suggestion
By default, Prometheus keeps 15 days of data. It may be useful to increase the history window to keep wider history. To do so, update the Prometheus service execution argument (typically, in `/etc/systemd/system/multi-user.target.wants/prometheus.service`).
```